2048.py

```diff
@@ -515,7 +500,11 @@ def draw_start_cheat():
 
     pg.display.update()
 
-def move1c(a,b):
+def move1c(start, increment, step):
```

Comment:
Try to simplify the other move*c functions in the same way

Reply:
I don't fully understand what you did. Do you think you could explain to me at lunch or something? My lunch is from 12:30 to 1:05.